content/Animations in React Native - Fernanda Toledo.md

```diff
@@ -0,0 +1,116 @@
+### Working with animations in React Native
+
+### The cover image
+
+Always try to include a cover image at the beginning of the post, as it helps catching the attention of the reader.
+Here are some links to find images: https://unsplash.com, https://www.pexels.com. Remember that they should be public.
```

Comment:
I guess this will be replaced, right? hahah

Reply:
Done :)